protocol/test/Unripe.test.js

```diff
@@ -72,21 +72,20 @@ describe('Unripe', function () {
 
     it('getters', async function () {
       expect(await this.unripe.getUnderlyingPerUnripeToken(UNRIPE_BEAN)).to.be.equal(to6('0.1'))
-      expect(await this.unripe.getPenalty(UNRIPE_BEAN)).to.be.equal(to6('0'))
-      expect(await this.unripe.getPenalizedUnderlying(UNRIPE_BEAN, to6('1'))).to.be.equal('0')
+      expect(await this.unripe.getPenalty(UNRIPE_BEAN)).to.be.equal(to6('0.01'))
+      expect(await this.unripe.getPenalizedUnderlying(UNRIPE_BEAN, to6('1'))).to.be.equal(to6('0.01'));
       expect(await this.unripe.getTotalUnderlying(UNRIPE_BEAN)).to.be.equal(to6('100'))
       expect(await this.unripe.isUnripe(UNRIPE_BEAN)).to.be.equal(true)
       expect(await this.unripe.getUnderlying(UNRIPE_BEAN, to6('1'))).to.be.equal(to6('0.1'))
       expect(await this.unripe.balanceOfUnderlying(UNRIPE_BEAN, userAddress)).to.be.equal(to6('100'))
-      expect(await this.unripe.balanceOfPenalizedUnderlying(UNRIPE_BEAN, userAddress)).to.be.equal('0')
+      expect(await this.unripe.balanceOfPenalizedUnderlying(UNRIPE_BEAN, userAddress)).to.be.equal(to6('10'))
     })
 
     it('gets percents', async function () {
       expect(await this.unripe.getRecapPaidPercent()).to.be.equal('0')
       expect(await this.unripe.getRecapFundedPercent(UNRIPE_BEAN)).to.be.equal(to6('0.1'))
       expect(await this.unripe.getRecapFundedPercent(UNRIPE_LP)).to.be.equal(to6('0.188459'))
-      expect(await this.unripe.getPercentPenalty(UNRIPE_BEAN)).to.be.equal(to6('0'))
-      expect(await this.unripe.getPercentPenalty(UNRIPE_LP)).to.be.equal(to6('0'))
+      expect(await this.unripe.getPercentPenalty(UNRIPE_BEAN)).to.be.equal(to6('0.01'))
```

Comment:
could we add the percent penalty for unripeLP as well?

Reply:
Since %penalty is no longer based on the RecapPaidPercentAmount or the RecapFundedPercent and we dont add any underlying for UNRIPE_LP before the test this should be 0. Added here  [6f02dd2](https://github.com/BeanstalkFarms/Beanstalk/pull/802/commits/6f02dd2019cec27a2f721b425e5ec1b1ad46edba)

Reply:
Figured out what was going on and adjusted here like we talked about [c2c3b95](https://github.com/BeanstalkFarms/Beanstalk/pull/802/commits/c2c3b9590064438449456809105e72a05798859f)